src/service/mobile_config.proto

```diff
@@ -213,13 +213,17 @@ enum admin_key_role {
   carrier = 3;
   // propagation calculation service of a mobile carrier
   pcs = 4;
+  // key for signing ban requests
+  admin_key_role_banning = 5;
```

Comment:
naming seams of? would it just not be `banning` to match he other names. or `admin_banning` to avoid the network enum collision?

Reply:
How do you feel about ?
```
admin_key_role.admin_banning;
network_key_role.network_banning;
```

Reply:
Because of the proto enum name elision thing. the rust struct does read as `AdminKeyRole::Banning`.

Reply:
I like clean rust names!

Reply:
If we want to be able to use `AdminKeyRole::Banning` and `NetworkKeyRole::Banning` in rust, then yes.

Reply:
Keep as is then